src/web/app/pages-instructor/instructor-session-edit-page/instructor-session-edit-page.component.html

```diff
@@ -20,6 +20,8 @@
                              (deleteCurrentQuestionEvent)="deleteExistingQuestionHandler(i)"
                              [(formModel)]="questionEditFormModels[i]"></tm-question-edit-form>
     </div>
+    <h6 align="right" style="color: #36B; margin-bottom: -3px; cursor: pointer;" (click)="scrollUp()" id="scrollUp">Back to top of page</h6>
```

Comment:
Can we change this to an anchor tag instead of a `h6` don't think a header is appropriate for this function. Let's try to also avoid inline styles and move this into the style sheet (Remember to remove any styles that aren't needed after this change)